src/Phinx/Db/Table.php

```diff
@@ -588,7 +588,8 @@ public function addTimestampsWithTimezone($createdAt = null, $updatedAt = null)
     public function insert($data)
     {
         // handle array of array situations
-        if (isset($data[0]) && is_array($data[0])) {
+        $firstKey = array_shift(array_keys($data));
```

Comment:
You can only pass variables to `array_shift` If you pass a function call, it results in an error

Reply:
Hmm, oke. Didn't saw any errors while running the tests. Whatever, easy change.